src/sinol_make/commands/run/structs.py

```diff
@@ -17,3 +17,13 @@ class ValidationResult:
 	changes: List[ResultChange]
 	expected_scores: dict
 	new_expected_scores: dict
+
+@dataclass
+class ExecutionResult:
+	# Result status of execution. Can be one of:
+	# "OK", "WA", "TL", "ML", "RE", "CE"
+	Status: str
+	# Time in milliseconds
+	Time: int
+	# Memory in KB
+	Memory: float
```

Comment:
It's the memory that should be in int, right? (but maybe the time also, which would make sense when it's in milliseconds)

Reply:
Usually it's enough to read the code in the GitHub interface before re-adding someone as a reviewer. The reviewer often just re-reads the whole code and if you're doing it first, it allows to skip a lot of going back-and-forth with the review.